src/main/java/com/hermesworld/ais/galapagos/adminjobs/impl/NoUpdatesAdminClient.java

```diff
@@ -46,22 +44,24 @@ public void close(Duration timeout) {
 
     @Override
     public CreateTopicsResult createTopics(Collection<NewTopic> newTopics) {
-        throw new UnsupportedOperationException();
+        return delegate.createTopics(Collections.singletonList(new NewTopic("test", Map.of())),
```

Comment:
No need to pass different topics here. Just pass `newTopics`, as it is still quite cool to have the server validate the set.